build_pdf.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Parse languages and ensure that "en" is built last
```

Comment:
Why does it matter that `en` is built last?

Reply:
Hmm, for PDF builds it actually does not matter. For website builds it does matter because we're generating the redirect rules in the same loop and en should be the fallback.

Reply:
I'll remove that code from the build_pdf.sh script.